res/css/views/dialogs/_ForwardDialog.scss

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2021 Robin Townsend.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_ForwardDialog {
+    width: 520px;
+    color: $primary-fg-color;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    min-height: 0;
+    height: 80vh;
+
+    > h3 {
+        margin: 0 0 6px;
+        color: $secondary-fg-color;
+        font-size: $font-12px;
+        font-weight: $font-semi-bold;
+        line-height: $font-15px;
+    }
+
+    > .mx_ForwardDialog_preview {
+        max-height: 30%;
+        flex-shrink: 0;
+        overflow: scroll;
+
+        div {
+            pointer-events: none;
```

Comment:
Given we disable interactions, we should probably get rid of things which entice it

![image](https://user-images.githubusercontent.com/2403652/120312113-38fca680-c2d0-11eb-9b86-ab2c93edad8a.png)


The thumbnail may want to be smaller too.

Reply:
Perhaps, as for thumbnails I've personally been waiting to see where #5682 goes, but restricting them here is something I could do

Reply:
> Given we disable interactions, we should probably get rid of things which entice it

On the other hand, those are arguably important to previewing how the message will actually look. A user might be confused if they see their video just look like a still image with no controls in the preview.

Reply:
I was talking about the `Download XYZ` but your argument for an accurate preview is valid

Reply:
Ah okay, the download button seems reasonable to hide, I'll do that